core/templates/pages/exploration-editor-page/ckeditor_dependency.html

```diff
@@ -0,0 +1,25 @@
+<!--Preload the styles required for ckeditor.-->
+<link rel="preload" href="/third_party/static/ckeditor-bootstrapck-1.0.0/skins/bootstrapck/editor.css?t=J5S9" as="style">
+<link rel="preload" href="/third_party/static/ckeditor-4.12.1/plugins/scayt/dialogs/dialog.css?t=J5S9" as="style">
+<link rel="preload" href="/third_party/static/ckeditor-4.12.1/plugins/tableselection/styles/tableselection.css" as="style">
+
+<!--Preload the ckeditor rte-components thumbnail.-->
+<link rel="preload" type="image/png" href="/extensions/ckeditor_plugins/pre/icons/pre.png?t=J5S9" as="image">
+<link rel="preload" type="image/png" href="/third_party/static/ckeditor-bootstrapck-1.0.0/skins/bootstrapck/icons_hidpi.png" as="image">
```

Comment:
@vojtechjelinek Only these two images will be imported as png because the second one is from a third-party library so we can not make any changes in the code of third party library and as for the first one the image URL is by default set to this. I tried searching a file, where this URL is set but I didn't find any. Most probably it is automatically set by a third-party library so I didn't change it to webp. Also, the difference between the sizes for these two images is only 2 KB so we can keep these as it is.